graylog2-web-interface/src/contexts/GlobalStylesProvider.jsx

```diff
@@ -1,6 +1,31 @@
-import { createGlobalStyle, css } from 'styled-components';
-
-const GlobalThemeStyles = createGlobalStyle(({ additionalStyles, theme }) => css`
+// @flow strict
+import * as React from 'react';
+import { createContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import { createGlobalStyle, css, type CSSRules } from 'styled-components';
+
+type Props = {
+  children: React.Node,
+};
+
+type StyleTypes = {
+  addGlobalStyles: (?CSSRules) => ?string,
+};
+
+export const GlobalStylesContext = createContext<StyleTypes>({ addGlobalStyles: () => {} });
+
+const GlobalStylesProvider = ({ children }: Props) => {
+  const [additionalStyles: string, addGlobalStyles] = useState();
```

Comment:
Let's unify this type with the type used in line 12 and define the type like this: `const [additionalStyles, addGlobalStyles] = useState<?string>();`